src/Draco.Editor.Web/app/ts/dotnet.ts

```diff
@@ -0,0 +1,103 @@
+import { downloadAssemblies } from './cache.js';
+
+const compilerWorker = new Worker('worker.js'); // first thing: we start the worker so it loads in parallel.
+let runtimeWorker: Worker | undefined;
+let listeners : ((arg: {outputType: string; value: string, clear: boolean}) => void)[] = [];
+let stdoutBuffer = 'Loading Compiler\'s .NET Runtime...';
+compilerWorker.onmessage = async (ev) => {
+    const msg = ev.data as {
+        type: string;
+        message: string;
+    };
+    switch (msg.type) {
+    case 'setOutputText': {
+        const parsed = JSON.parse(msg.message);
+        onOutputChange(parsed['OutputType'], parsed['Text'], true);
+    }
+        break;
```

Comment:
Indentation wrong compared to braces. Maybe just move within the braces?

Reply:
It's wtf the formatter want, yeah i'll move it in